skxray/fitting/base/parameter_data.py

```diff
@@ -136,4 +137,189 @@ def get_para():
        based on different algorithms.
        Use copy for dict.
     """
-    return para_dict
\ No newline at end of file
+    return para_dict
+
+default_param = {
```

Comment:
this dictionary should go into a config file that gets packaged with scikit-xray